projects/ngx-sortablejs/package.json

```diff
@@ -18,9 +18,9 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@angular/common": "^11.0.0",
-    "@angular/core": "^11.0.0",
-    "sortablejs": ">=1.7.0"
+    "@angular/common": "^13.0.0",
```

Comment:
I think for @angular need more wide range, for example >=12.0.0 because it compatible. Because soon 14 version will be released and again we will need to update package.json.

Reply:
Even if we change it to `"@angular/core": ">=13.0.0"`, it will stop working when Angular 15 comes out because Angular only keeps backwards compatibility for 2 major versions. Furthermore, we should upgrade `package.json` when Angular 14 comes out anyway, because it improves package compilation time and size, especially for users who like to stay up-to-date with Angular core.

Besides, Angular recommends that libraries keep major version in line with Angular, so the users can simply install `ngx-sortablejs@13` if they're using Angular 13, or `ngx-sortablejs@14` if they're using Angular 14.

Reply:
Angular supports last 2 major version with updates, but support libraries compiled without ivy if they doesn't use specific new features, and some of this libraries compiled with angular 7 version as for me.